### PR TITLE
Update set-up.md

### PR DESCRIPTION
**Pull request by neillnolann**
- [Microsoft365.com/setup](https://soft365setup.com/)
- [Set up Microsoft 365 Business Basic](setup-business-basic.md)
- [Set up Microsoft Business Standard](setup-business-standard.md)
- [Set up Microsoft 365 Business Premium](../../business/set-up.md)
- [Set up Microsoft 365 Apps for business](setup-apps-for-business.md)

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/business-video/set-up.md [hunk 1]
```diff
@@ -42,4 +42,11 @@ description: "Learn how to set up Microsoft 365 Business Premium."
 12. On the  **Protect data in Office for mobile**  page, leave mobile app management on, expand the settings and review them, and then select  **Create mobile app management policy**.
 13. After your setup is complete, return to the Microsoft admin center. In the admin center you can continue setting up features like Windows 10 policies, DLP, etc. on the  **Setup**  page.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/set-up.md:43]:
```suggestion
13. After your setup is complete, return to the Microsoft 365 admin center. In the admin center, you can continue setting up features like Windows 10 policies, DLP, etc. on the  **Setup**  page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - two additional minor edits, please.

**Comment by yogkumgit** [comment]
@neillnolann Please implement the suggestions. Thanks

**Comment by pebaum** [comment]
no way. this Microsoft365.com/setup link actually links to https://soft365setup.com/ a phishing site.

**Pull request closed without merging**